src/client/src/modals/CosmosDbModal/messages.ts

```diff
@@ -17,10 +17,6 @@ const messages = defineMessages({
     id: "CosmosModal.hideAdvancedMode",
     defaultMessage: "Hide advanced mode",
   },
-  ariaCloseModalLabel: {
```

Comment:
maybe worth for accessibility keep the customisation of the messages on the modals as the title